README.md

```diff
@@ -4,44 +4,51 @@
 
 The educandu framework
 
+## Perequisites
+
+ * Node 16 or higher
+ * Docker
+ * gulp installed globally (for conveninence): npm i -g gulp
+
 ## How to use
 
 ~~~
 $ yarn add @educandu/educandu
 ~~~
 
-Use it in code like this:
+Use it in code as follows:
 
 ~~~
 import educandu from '@educandu/educandu';
 
 educandu({
-  port: Number(process.env.PORT) || 3000,
-  mongoConnectionString: 'mongodb://user:pwd@localhost:27017/my-db?replicaSet=myrs&authSource=admin',
-  skipMongoMigrations: process.env.SKIP_MONGO_MIGRATIONS === 'true',
-  skipMongoChecks: process.env.SKIP_MONGO_CHECKS === 'true',
+  port: 3000, //on which to run the application
+  mongoConnectionString: 'mongodb://root:rootpw@localhost:27017/dev-educandu-db?replicaSet=educandurs&authSource=admin',
+  skipMongoMigrations: process.env.TEST_APP_SKIP_MONGO_MIGRATIONS === true.toString(),
+  skipMongoChecks: process.env.TEST_APP_SKIP_MONGO_CHECKS === true.toString(),
   cdnEndpoint: 'http://localhost:9000',
   cdnRegion: 'eu-central-1',
-  cdnAccessKey: 'KSHJFHKJHFSGDJVAJHD',
-  cdnSecretKey: 'ZUUUHFJKJKHDJKHJKKJhdkjsdhku',
-  cdnBucketName: 'my-cdn-bucket',
-  cdnRootUrl: 'http://localhost:9000/my-cdn-url',
-  sessionSecret: '49zcn238zt43zt7c8234nt8843t8',
-  sessionDurationInMinutes: Number(process.env.SESSION_DURATION_IN_MINUTES) || 60,
-  smtpOptions: {
-    host: 'localhost',
-    port: 8025,
-    ignoreTLS: true
-  },
-  publicFolders: [path.resolve('./assets')],
+  cdnAccessKey: 'UVDXF41PYEAX0PXD8826',
+  cdnSecretKey: 'SXtajmM3uahrQ1ALECh3Z3iKT76s2s5GBJlbQMZx',
+  cdnBucketName: 'dev-educandu-cdn',
+  cdnRootUrl: 'http://localhost:9000/dev-educandu-cdn',
+  sessionSecret: 'd4340515fa834498b3ab1aba1e4d9013',
+  sessionDurationInMinutes: 60,
+  emailSenderAddress: 'educandu-test-app@test.com',
+  smtpOptions: 'smtp://localhost:8025/?ignoreTLS=true',
+  publicFolders: ['./test-app/dist', './test-app/static'].map(x => path.resolve(x)),
+  resources: ['./test-app/resource-overrides.json'].map(x => path.resolve(x)),
   initialUser: {
     username: 'test',
     password: 'test',
     email: 'test@test.com'
-  }
+  },
+  exposeErrorDetails: true //set to false in order to hide error details such as stack trace on productive systems
 });
 ~~~
 
+This should run both the entire application and provide you with an admin user as defined on the initialUser setting. In order to create other admins please consult the users page in the application. Here, admins can promote other users to the admin role.
```

Comment:
I don't think this is currently possible

Reply:
Creating other admins?

Reply:
Yep, not possible on the UI currently

Reply:
Wow, yes, you are right... I deleted that part.